README.md

```diff
@@ -12,10 +12,17 @@ This project is an implementation of Uniswap V2, a decentralized exchange protoc
 - If you're on Windows, you will need to run this command in  Git BASH or WSL: 
  `curl -L https://foundry.paradigm.xyz | bash`
 
-- to initialize a new project : 
+- To initialize a new project : 
  `forge init `
+
```

Comment:
you can remove that part since your project is already initialized. readers of your project only care about how to compile (`forge build`) and how to test (`forge test`)